LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/contain-intrinsic-size-032-expected.txt

```diff
@@ -20,11 +20,11 @@ FAIL .test 6 assert_equals:
 clientWidth expected 0 but got 34
 FAIL .test 7 assert_equals:
 <select class="test ciw-100 cih-none" data-expected-client-width="100" data-expected-client-height="18"></select>
```

Comment:
The failures in this test are related to https://bugs.webkit.org/show_bug.cgi?id=246338